configs/eslint-config-mongosh/index.js

```diff
@@ -93,6 +93,7 @@ module.exports = {
         ...common.testRules,
         ...extraJSRules,
         ...extraTypescriptRules,
+        '@typescript-eslint/no-non-null-assertion': 'off',
```

Comment:
Just going to mention that I think that this is on almost everywhere in our repos and that hasn't been a major issue for the most part